packages/api/modules/posts/procedures/index.ts

```diff
@@ -0,0 +1,180 @@
+import { z } from 'zod';
+import { router, publicProcedure } from '../../../trpc/base';
+
+export const postsRouter = router({
+  create: publicProcedure
+    .input(z.object({
+      content: z.string().min(1),
+      authorId: z.string(),
+    }))
+    .mutation(async ({ input, ctx }) => {
+      try {
+        const post = await ctx.prisma.post.create({
+          data: {
+            content: input.content,
+            authorId: input.authorId,
+          },
```

Comment:
suggestion: esto podría resumirse a una linea. (lo mismo en la mayoría de los procedures)
```suggestion
          data: input,
```